lib/telemetry_metrics_statsd/formatter/basic.ex

```diff
@@ -0,0 +1,30 @@
+defmodule TelemetryMetricsStatsd.Formatter.Basic do
```

Comment:
Is this name acceptable?

Reply:
I'm not sure, to me basic kinda suggests that it's inferior to others, or incomplete 😄 We could use "Default", but it is used by default anyway, so this name wouldn't make much sense. Did you think about alternatives? If no, let's stick with basic.

Reply:
What about `Standard`? AFAIK is the StatsD standard format.

Looking forward to seeing this pull request merged because I also need the DataDog format :)

Reply:
Standard is great, actually

Reply:
Yup, sounds good :)